sdk/communication/Azure.Communication.Common/src/PhoneNumberIdentifier.cs

```diff
@@ -25,6 +28,33 @@ public override string RawId
         /// <summary>The phone number in E.164 format.</summary>
         public string PhoneNumber { get; }
 
+        /// <summary>True if the phone number is anonymous, e.g. when used to represent a hidden caller Id.</summary>
+        public bool IsAnonymous => RawId == $"{Phone}{Anonymous}";
+
+        /// <summary>The asserted Id is set on a phone number that is already in the same call to distinguish from other connections made through the same number.</summary>
+        public string AssertedId
+        {
+            get
+            {
+                if (_assertedId == "")
```

Comment:
maybe we can use `string.IsNullOrEmpty` so also check for null

Reply:
reverted